app/routes/router.js

```diff
@@ -92,6 +92,16 @@ router
     .get('/findalltransactionsbyparamid', function (req, res){
         API.findTransactionsByParamId(req, res)
     })
+	
+	// gets all the errors
+    .get('/findallerrorreports', function (req, res){
```

Comment:
@seancavanaugh Yes it is just a code quality issue, but I think we should be shooting for high quality code. Following standard REST practices, the endpoint url should represent the object being interacted with, and the query would specify whether we are getting all data or filtering. I know it really isn't that big of a deal, but small things like this still shouldn't be overlooked.